src/Jenssegers/Mongodb/Relations/EmbedsOne.php

```diff
@@ -28,6 +28,17 @@ public function getResults()
         return $this->toModel($this->getEmbedded());
     }
 
+    /**
+     * @inheritdoc
```

Comment:
Delete this doc block, please

Reply:
Erm.. may I ask why?

Reply:
It is useless doc block

Reply:
Well.. `inheritdoc` is better then no block, considering this method is a override. 

Proceed with the removal?

Reply:
Better type hints and doc blocks for params and return type  if type hints don't exist. Doc block `inheritdoc` seems like default useless doc block

Reply:
Again why redoc when this is an override? This is not the default block `getEager()` is a method from Eloquent Builder.

Reply:
I think it's better to keep it as it is.

Reply:
Using `inheritdoc` don't contain benefits here